recipes/gtest/all/conanfile.py

```diff
@@ -18,6 +18,27 @@ class GTestConan(ConanFile):
     default_options = {"shared": False, "build_gmock": True, "fPIC": True, "no_main": False, "debug_postfix": 'd', "hide_symbols": False}
     _source_subfolder = "source_subfolder"
 
+    @property
+    def _minimum_cpp_standard(self):
+        if self.version == "1.8.1":
+            return 98
+        else:
+            return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        if self.version == "1.8.1":
+            return {
+                "Visual Studio": "13"
+            }
+        else:
+            return {
+                "Visual Studio": "15",
+                "gcc": "5",
+                "clang": "5",
+                "apple-clang": "9.1",
```

Comment:
Are you sure? It worked in 1.10.0 without these checks. Visual Studio 2017 seems excessive for C++11, gcc 5 and clang 5 (clang 3.4 has wide support of C++14).

Reply:
These are the compilers officially supported by gtest. Note that only a warning is given, not an error.

Ref. https://github.com/google/googletest#compilers

Reply:
Visual Studio 2015 = Visual Studio 14

Reply:
Fixed, I'm not sure what we should do with 1.10.0 as it did work I guess, but according to the maintainer it should not...